README.md

```diff
@@ -69,60 +71,98 @@ This section explains how to install the latest stable release of
 
-Using one (and only one) of these scientific Python distributions:
+If in doubt, you should install via this method; it generally has the
+least likelihood of potential pitfalls for non-experts, and we may be
+able to provide limited assistance if you do run into trouble.
 
-1. [Anaconda](http://continuum.io/downloads)
-2. [WinPython](https://winpython.github.io/)
-3. [Python(x,y)](http://python-xy.github.io)
+### The Harder Way: Alternative Distributions
 
-### On Mac OSX:
+**Important Note:** While we offer alternative options for users who
+desire them, if you encounter installation issues you are unable to
+solve on your own, we recommend you use Anaconda instead, as we are
+generally unable to offer individual assistance for problems specific
+to installing via these alternative distributions.
 
-- Using the [Anaconda Distribution](http://continuum.io/downloads).
-- Through [MacPorts](http://www.macports.org/).
+**Windows**
 
-### On GNU/Linux
+Spyder is included in these other scientific Python distributions:
 
-- Through your distribution package manager (i.e. `apt-get`, `yum`,
-  etc).
-- Using the [Anaconda Distribution](http://continuum.io/downloads).
-- Installing from source (see below).
+* [WinPython](https://winpython.github.io/)
+* [Python(x,y)](http://python-xy.github.io)
 
-### Cross-platform way from source
+**macOS**
 
-You can also install Spyder with the `pip` package manager, which comes by
-default with most Python installations. For that you need to use the
-command:
+Spyder can be obtained through through
+[MacPorts](http://www.macports.org/).
 
-    pip install spyder
+**GNU/Linux**
 
-To upgrade Spyder to its latest version, if it was installed before, you need
-to run
+* You can often get Spyder through your distribution's package
+manager (i.e. `apt-get`, `yum`, etc), or install from source
+(see below).
 
-    pip install --upgrade spyder
+### The Expert/Cross-platform Way: Installing from source with pip
 
-For more details on supported platforms, please refer to our
-[installation instructions](http://pythonhosted.org/spyder/installation.html).
+**Warning:** While this installation method is a viable option for
+experienced users, installing Spyder (and other SciPy stack packages)
+with `pip` can encounter a number of tricky issues. While you are welcome
+to try this on your own if you're an expert, we are unable to help if you
+do run into problems, except to recommend using Anaconda instead.
+
+You can also install Spyder with the `pip` package manager, which comes by
+default with most Python installations. To do so, you need to use the command:
+```
+    pip install spyder
+```
+To upgrade Spyder to its latest version, if it was installed before, you need
+to run:
+```
+    pip install --upgrade spyder
+```
 
 **Important note**: This does not install the graphical Python libraries (i.e.
 PyQt5 or PyQt4) that Spyder depends on. Those have to be installed separately
 after installing Python.
```

Comment:
This is only true now for Python 2, so please reword this note.

Reply:
Already done